backstop/tests/table.html

```diff
@@ -273,15 +273,66 @@ <h2>Default</h2>
             </div>
             <div class="iui-cell iui-slot">
               <button
-                class="iui-button iui-borderless"
+                class="iui-button iui-borderless iui-dropdown"
                 aria-label="Modify columns"
                 type="button"
+                onclick="openColumnManger()"
               >
                 <svg-column-manager
                   class="iui-button-icon"
                   aria-hidden="true"
                 ></svg-column-manager>
               </button>
+              <div
+                data-tippy-root=""
+                id="tippy-1"
+                style="z-index: 9999; display: none; position: fixed; inset: 0px auto auto 0px; margin: 0px; transform: translate(625px, 62px);"
```

Comment:
Positioning is messed up. You _can_ hardcode the numbers but you should base it relative to the trigger button rather than the screen. These magic numbers will never work consistently across different screens.

Also, are you planning to add the column manager to other table instances on this page? Because then you might want to calculate the position using JS.